sinks/types.go

```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 )
 
-var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery]")
+var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery | gcm ]")
```

Comment:
Do we need any other auth information for gcm? What will the user have to do to get GCM to work?